lib/impure/rdstdin.nim

```diff
@@ -67,3 +67,20 @@ else:
     linenoise.free(buffer)
     result = true
 
+template ask*(question: string, default: bool): bool =
+  ## Convenience template for ``readLineFromStdin`` and ``parseBool`` inside a ``try`` block,
+  ## to ask a question to the user on the terminal and return a boolean value.
+  ## You can provide a default boolean value that will be returned when ``parseBool``
+  ## can not parse the user input, this template does not raise ``ValueError`` by itself.
+  ## A postfix string can be appended at the end of the question.
+  ## If verbose is ``true`` the response will be printed to the terminal.
```

Comment:
Also, after the last commit, there's no `verbose` nor `postfix` anymore.

Reply:
@Araq say to remove the 3 default parameters.
I think is a lot more useful with default parameters, because is a DRY template.
Also I dont understand whats the problem with >=3 default parameters. :woman_shrugging: 
I think is a lot more useful with `parseBool` to parse bools.

This kind of terminal question is seen everywhere with the `"FOO? (y/n): "`, 
so is not like a weird concept or something.